memdx/ops_core.go

```diff
@@ -9,25 +9,32 @@ import (
 type OpsCore struct {
 }
 
-func (o OpsCore) decodeErrorContext(resp *Packet, err error) error {
+func (o OpsCore) decodeErrorContext(resp *Packet, err error, dispatchedTo string, dispatchedFrom string) error {
```

Comment:
It might be useful to just pass the Dispatcher to this method, so we can capture whatever state we want in one place (DRY).  But this also works.

Reply:
I'm going to merge this as is, just because I think that passing in an interface with a Dispatch() call would be a bit weird.